### PR TITLE
Feature/update google maps

### PR DESCRIPTION
**Pull request by JoeKer1**
Updated to latest version of angular google maps. Everything seems to work with the agm/js-clusterer package as well. Should we still use our directive or this one?

### CODE REVIEW

File: src/app/components/map/map.component.ts [hunk 1]
```diff
@@ -24,6 +24,14 @@ import { MapView } from '../../models/map-view';
 export class MapComponent implements OnInit {
 
   @Input() searchResults: PinSearchResultsDto;
+  private styles = [{
+          url: '//crds-cms-uploads.s3.amazonaws.com/connect/CLUSTER.svg',
+          height: 50,
+          width: 50,
+          textColor: '#fff'
+        }];
```
Comment (vchtelmakh) [inline src/app/components/map/map.component.ts]: Probably a const, but ¯\_(ツ)_/¯

File: src/app/components/map/map.component.ts [hunk 2]
```diff
@@ -24,6 +24,14 @@ import { MapView } from '../../models/map-view';
...
+  private maxZoom = 17;
```
Comment (harimau777) [inline src/app/components/map/map.component.ts]: The maximum we use when calculating the zoom is 15, do we want to use the same value here for the maximum the user can manually zoom out?

Should any of these settings be added to the constants file?

File: src/app/components/map/map.component.ts [hunk 3]
```diff
@@ -24,6 +24,14 @@ import { MapView } from '../../models/map-view';
 export class MapComponent implements OnInit {
 
   @Input() searchResults: PinSearchResultsDto;
+  private styles = [{
```
Comment (harimau777) [inline src/app/components/map/map.component.ts]: Should we/can we add this to one of the css files?

File: src/app/components/map/map.component.ts [hunk 4]
```diff
@@ -24,6 +24,14 @@ import { MapView } from '../../models/map-view';
...
+          url: '//crds-cms-uploads.s3.amazonaws.com/connect/CLUSTER.svg',
+          height: 50,
+          width: 50,
+          textColor: '#fff'
+        }];
+  private maxZoom = 17;
+  private averageCenter = true;
```
Comment (harimau777) [inline src/app/components/map/map.component.ts]: The AOT requires properties that are used by a template to be public.  Do we want to start doing this even though the AOT isn't ready yet?

File: src/app/components/map/map.component.ts (hunk 1 above)
Comment (JoeKer1) [reply src/app/components/map/map.component.ts]: Yeah I just wanted to get it working first :D

File: src/app/components/map/map.component.ts (hunk 2 above)
Comment (JoeKer1) [reply src/app/components/map/map.component.ts]: I just used what was already there in our directive, I'm not sure what it should be. Should probably be in constants though.

File: src/app/components/map/map.component.ts (hunk 4 above)
Comment (JoeKer1) [reply src/app/components/map/map.component.ts]: I read an issue last night that stated that none of the angular google maps packages support AoT yet, but that's probably not a bad idea if we have to do it for AoT in the future.

Comment (JoeKer1) [reply src/app/components/map/map.component.ts]: https://github.com/SebastianM/angular-google-maps/issues/1143

**Comment by JoeKer1** [comment]
Right now  this breaks labels. So we may need to go back to our old clusterer for now.

**Pull request merged**